themes/bootstrap3/js/common.js

```diff
@@ -696,3 +696,25 @@ $(function commonDocReady() {
 
   setupIeSupport();
 });
+
+$(function searchFormResetHandler() {
+  const queryInput = $('#searchForm_lookfor');
+  const resetButton = $('#searchForm-reset');
+
+  let query = queryInput.val();
+  if (query !== '') {
+    resetButton.show();
+    queryInput.focus().val('').val(query);
+  }
+  queryInput.on('input', function onInput() {
+    if ($(this).val() === '') {
+      resetButton.hide();
+    } else {
+      resetButton.show();
+    }
+  });
+  resetButton.on('click', function onClick() {
+    queryInput.attr('value', '').focus();
+    resetButton.hide();
+  });
+});
```

Comment:
We try to have an end-of-file line break in all of our files for style purposes. Probably worth adding one back here.

Reply:
Line break added